### ISSUE TITLE
Add `ignore: ["keyframe-selectors"]` to `rule-selector-property-disallowed-list`

### ISSUE THREAD

**Issue by emmacharp** [stylelint/stylelint#7511]
### What minimal example or steps are needed to reproduce the bug?

```css
@keyframes fade-in {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}
```
### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "rule-selector-property-disallowed-list": [{
        "/^[a-z]+$/": ["opacity"]
      }
    ]
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBrGBPAZgJwIYFsYBnAAk11gFoBLAOxOAB16zsJ8HmTuSIAHXGGoAXdPBIAGANxcSAX2azhETi279BIsSQCMMlgtpyQAGhCZqAGxgA5AnEQwAHgT7WAdGCJFT4CLQsAcwQQJhZGEGwAV2siCPEwnhII6OtKIhhrMGVsSj42PhhsUUooaiJcS0sIAHcYKEpLcuF4kgBtRKSeCIB6AD023EoALwBdAGoAEh7WtoiNIVEI0dkkwySVg2ZjM0gA6kCAMQhsfFxhEIArIn9fWD4fRE6IolFrJtoWhGSQS3PiL4mWQvN6ZOjCSh7ILpYS4WhQXDYKCtCJ-YQAiLbEByIA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problem was reported: ```Unexpected property "opacity" for selector "from" [3:5-12]```

### Do you have a proposal to fix the bug?

Maybe the same fix as this ? #7417

**Comment by ybiquitous** [comment]
@emmacharp Thanks for the proposal with the reproducible demo. Adding the new option sounds good to me. 👍🏼 

> Maybe the same fix as this ? #7417

I think so. The suggested new option can be implemented as the [`ignore: ["keyframe-selectors"]` option of `selector-disallowed-list`]( https://stylelint.io/user-guide/rules/selector-disallowed-list/#ignore-keyframe-selectors).

If no one else objects, let's implement it.

**Comment by emmacharp** [comment]
@ybiquitous, can we expect this fix to be included in the 16.3.0 release?

**Comment by ybiquitous** [comment]
Sorry, I missed the issue. I've labeled the issue as **ready to implement**. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to add a new option](https://stylelint.io/developer-guide/rules#add-an-option-to-a-rule) in the Developer guide.

> can we expect this fix to be included in the 16.3.0 release?

If anyone implements it soon, it's possible 👍🏼

**Comment by emmacharp** [comment]
Thanks.

As I said, not that much of a programmer myself but I'll see if I can contribute, I have time this week.
